pages.pl/linux/vipw.md

```diff
@@ -0,0 +1,12 @@
+# vipw
+
+> Edytuje plik password.
```

Comment:
I am not sure about this command shouldn't this be the same as line 6 `Edycja pliku password:`

Reply:
I understand your concert, but IMHO it's better this way.
First one is "what command does?" it's like "it edits the file"
The second one is more like "editing the file" or "edition of the file"